python/src/ops.cpp

```diff
@@ -3645,6 +3645,42 @@ void init_ops(nb::module_& m) {
         Returns:
             array: ``alpha * (a @ b)  + beta * c``
       )pbdoc");
+  m.def(
+      "block_masked_mm",
+      &block_masked_mm,
+      nb::arg(),
+      nb::arg(),
+      "tile_size"_a = 64,
+      "mask_out"_a = nb::none(),
+      "mask_lhs"_a = nb::none(),
+      "mask_rhs"_a = nb::none(),
```

Comment:
I might suggest we make these keyword only. But I don't feel too strongly about it. Forces the calling code more readable though.